### ISSUE TITLE
Image of text should use text - ordered list in graphic

### ISSUE THREAD

**Issue by ChrisXKeroack** [intel/openfl#857]
**Describe the bug**
Image of text should use text - ordered list in graphic

**To Reproduce**
1. Go to either https://openfl.readthedocs.io/en/latest/running_the_federation.html or https://openfl.readthedocs.io/en/latest/running_the_federation_with_gandlf.html
2. start a screenreader
3. Scroll down to graphic containing ordered list

notice: ordered list as image of text, should be represented with text; screenreader won't find and read the list that sighted user can see

**Expected behavior**
remove the image of text and present that ordered list in text as an ordered list

**Screenshots**
![image](https://github.com/securefederatedai/openfl/assets/104644211/4e8c6760-49c7-4201-a9f4-62de6a41da16)

**Desktop:**
This issue is not OS or browser specific, but for tracking purposes, testing was done on Win11, in Edge Version 114.0.1823.58 (Official build) (64-bit), and the screenreader in use was the common 3rd party screenreader NVDA (http://www.nvaccess.org).

**Smartphone:**
This issue is not device, OS, or browser specific.

**Additional context**
This is an accessibility issue, please annotate this bug with the 'accessibility' label.
